counting_sort/counting_sort.py

```diff
@@ -0,0 +1,25 @@
+def count_occurences(a, c):
+    for i in a:
+        c[a[i]] += 1
+    for i in range(1, len(c)):
+        c[i] = c[i - 1] + c[i]
+
+
+def counting_Sort(a, k):
+    b = [0] * len(a)
+    c = [0] * (k + 1)
+    countOccurences(a, c)
```

Comment:
It seems you didn't test your code before submitting the changes, please always check before updating.